### ISSUE TITLE
tabs: URL referencing

### ISSUE THREAD

**Issue by casperdcl** [iterative/cml.dev#123]
Given:

```
# /some/page.md
<toggle>
<tab title="Foo">
</tab>
<tab title="Bar">

some text

### sub section

</tab>
</toggle>
```

Should be able to use:

- `/some/page#Bar` to auto-open the appropriate tab (p1-important)
- `/some/page#Bar-sub-section` to auto-open the tab and scroll to containing section (p2-nice-to-have)

**Comment by casperdcl** [comment]
@julieg18 yes

**Comment by julieg18** [comment]
>  remember the ?tab=<arg> when navigating to a different page (p2-nice-to-have)

Are we sure we want this behavior? Currently it seems like we don't have many similiar tab names minus github, gitlab, and   bitbucket 🤔

**Comment by casperdcl** [comment]
> Are we sure we want this

yes, the GH/GL/BB tabs are important and frequent enough to make this desirable on https://cml.dev/doc. Potentially could clear `?tab=X` only when navigating to a page that doesn't contain an `X` tab.
